src/config/dynamic-theme-fixes.config

```diff
@@ -24643,6 +24643,15 @@ body,
 
 ================================
 
+temu.com
+
+CSS
+._2Fkk_bmp {
+    background-color: ${#fff0} !important;
```

Comment:
Did you mean to just use `transparent`? The 4-th `0` means zero opacity.

Reply:
I see the element has `background: hsla(0,0%,100%,0)`, which is equivalent to `background-color: #fff0`. Based on the contributions document, I should put that color inside `${}` since we want it inverted. 
But yes, effectively it is `transparent`. I actually did not know that keyword existed. Putting it inside `${}` or not yields the same result:
- If I do `background-color: transparent !important`, when Dark Reader is on, the result will be `background-color: transparent !important`. 
- If I do `background-color: ${transparent} !important`, when Dark Reader is on, the result will be `background-color: rgba(255, 255, 255, 0) !important`.

Reply:
It is interesting that the original CSS value of `hsla(0,0%,100%,0)` already means that the background color is transparent, but the computed value disregards the alpha component of the color. It stays the same when turning Dark Reader on, hence the problem. Forcing it to use `transparent` using the Dark Reader Dev Tools solves the problem for some reason. 

![image](https://github.com/darkreader/darkreader/assets/9978161/bb7d7ef0-fedf-4470-86da-d42ef30f358b)

Reply:
Yes, I meant `transparent` without `${}`. Those colors with 0 transparency may be useful for gradients (to fade out a white color for example), but not for usual backgrounds.